cypher/cypher-docs/src/docs/dev/ql/administration/security/database/admin-role-database.asciidoc

```diff
@@ -1,7 +1,41 @@
-As described in the section on <<administration-security-subgraph, sub-graph security>>, the `GRANT` command allows an administrator to grant a privilege to a role in order to access an entity.
-The `DENY` command allows an administrator to deny a privilege to a role in order to prevent access to an entity.
-The `REVOKE` command allows an administrator to remove a previously granted or denied privilege.
-The syntax is:
+The administrators can use the following Cypher commands to manage Neo4j database administrative rights.
+The components of the database privilege commands are:
+
+* the command:
+** `GRANT` – gives privileges to roles.
+** `DENY` – denies privileges to roles.
+** `REVOKE` – removes granted or denied privilege from roles.
+
+
+* _database-privilege_
+** `ACCESS` - allows access to a specific database.
+** `START` - allows the specified database to be started.
+** `STOP` - allows the specified database to be stopped.
+** `CREATE INDEX` - allows indexes to be created on the specified database.
+** `DROP INDEX` - allows indexes to be deleted on the specified database.
+** `INDEX [MANAGEMENT]` - allows indexes to be created and deleted on the specified database.
+** `CREATE CONSTRAINT` - allows constraints to be created on the specified database.
+** `DROP CONSTRAINT` - allows constraints to be deleted on the specified database.
+** `CONSTRAINT [MANAGEMENT]` - allows constraints to be created and deleted on the specified database.
+** `CREATE NEW [NODE] LABEL` - allows labels to be created so that future nodes can be assigned them.
+** `CREATE NEW [RELATIONSHIP] TYPE` - allows relationship types to be created, so that future relationships can be created with these types.
+** `CREATE NEW [PROPERTY] NAME` - allows property names to be created, so that nodes and relationships can have properties with these names assigned.
+** `NAME [MANAGEMENT]` - allows all of the name management capabilities: node labels, relationship types, and property names.
+** `ALL [[DATABASE] PRIVILEGES]` - allows access, index, constraint, and name management for the specified database.
+** `SHOW TRANSACTION {* | user[, ...]}` -  allows listing transactions and queries for the specified users on the specified database.
+** `TERMINATE TRANSACTION {* | user[, ...]}` - allows ending transactions and queries for the specified users on the specified database.
+** `TRANSACTION [MANAGEMENT] {* | user[, ...]}` - allows listing and ending transactions and queries for the specified users on the specified database.
+
+* _dbname_
+** The database to associate the privilege with.
+NOTE:: If you delete a database and create a new one with the same name, the new one will NOT have the privileges assigned to the deleted database.
```

Comment:
This note isn't rendering quite right. To sort it, replace line 31 with:

```
+
[NOTE]
====
If you delete a database and create a new one with the same name, the new one will NOT have the privileges assigned to the deleted database.
====
```

Note that you must include the `+` on line 31.